core/src/test/scala/com/databricks/labs/remorph/parsers/common/FunctionBuilderSpec.scala

```diff
@@ -191,243 +191,248 @@ class FunctionBuilderSpec extends AnyFlatSpec with Matchers with TableDrivenProp
+      ("RTRIM", FunctionDefinition.standard(1)),
+      ("SCHEMA_ID", FunctionDefinition.standard(0, 1)),
+      ("SCHEMA_NAME", FunctionDefinition.standard(0, 1)),
+      ("SCOPE_IDENTITY", FunctionDefinition.standard(0)),
+      ("SERVERPROPERTY", FunctionDefinition.standard(1)),
+      ("SESSION_CONTEXT", FunctionDefinition.standard(1, 2)),
+      ("SESSION_USER", FunctionDefinition.standard(0)),
+      ("SESSIONPROPERTY", FunctionDefinition.standard(1)),
+      ("SIGN", FunctionDefinition.standard(1)),
+      ("SIN", FunctionDefinition.standard(1)),
+      ("SMALLDATETIMEFROMPARTS", FunctionDefinition.standard(5)),
+      ("SOUNDEX", FunctionDefinition.standard(1)),
+      ("SPACE", FunctionDefinition.standard(1)),
+      ("SQL_VARIANT_PROPERTY", FunctionDefinition.standard(2)),
+      ("SQRT", FunctionDefinition.standard(1)),
+      ("SQUARE", FunctionDefinition.standard(1)),
+      ("STATS_DATE", FunctionDefinition.standard(2)),
+      ("STDEV", FunctionDefinition.standard(1)),
+      ("STDEVP", FunctionDefinition.standard(1)),
+      ("STR", FunctionDefinition.standard(1, 3)),
+      ("STRING_AGG", FunctionDefinition.standard(2, 3)),
+      ("STRING_ESCAPE", FunctionDefinition.standard(2)),
+      ("STUFF", FunctionDefinition.standard(4)),
+      ("SUBSTR", FunctionDefinition.standard(2, 3)),
+      ("SUBSTRING", FunctionDefinition.standard(2, 3)),
+      ("SUM", FunctionDefinition.standard(1)),
+      ("SUSER_ID", FunctionDefinition.standard(0, 1)),
+      ("SUSER_NAME", FunctionDefinition.standard(0, 1)),
+      ("SUSER_SID", FunctionDefinition.standard(0, 2)),
+      ("SUSER_SNAME", FunctionDefinition.standard(0, 1)),
+      ("SWITCHOFFSET", FunctionDefinition.standard(2)),
+      ("SYSDATETIME", FunctionDefinition.standard(0)),
+      ("SYSDATETIMEOFFSET", FunctionDefinition.standard(0)),
+      ("SYSTEM_USER", FunctionDefinition.standard(0)),
+      ("SYSUTCDATETIME", FunctionDefinition.standard(0)),
+      ("TAN", FunctionDefinition.standard(1)),
+      ("TIMEFROMPARTS", FunctionDefinition.standard(5)),
+      ("TODATETIMEOFFSET", FunctionDefinition.standard(2)),
+      ("TOSTRING", FunctionDefinition.standard(0)),
+      ("TRANSLATE", FunctionDefinition.standard(3)),
+      ("TRIM", FunctionDefinition.standard(1, 2)),
+      ("TYPE_ID", FunctionDefinition.standard(1)),
+      ("TYPE_NAME", FunctionDefinition.standard(1)),
+      ("TYPEPROPERTY", FunctionDefinition.standard(2)),
+      ("UNICODE", FunctionDefinition.standard(1)),
+      ("UPPER", FunctionDefinition.standard(1)),
+      ("USER", FunctionDefinition.standard(0)),
+      ("USER_ID", FunctionDefinition.standard(0, 1)),
+      ("USER_NAME", FunctionDefinition.standard(0, 1)),
+      ("VALUE", FunctionDefinition.xml(2)),
+      ("VAR", FunctionDefinition.standard(1)),
+      ("VARP", FunctionDefinition.standard(1)),
+      ("XACT_STATE", FunctionDefinition.standard(0)),
+      ("YEAR", FunctionDefinition.standard(1)))
 
```

Comment:
Why are you changing this in this PR? And why does it need to change?

Reply:
This is because I discovered bugs (fixed in other PRs) and pre-merged the corresponding PRs in order for the tests to pass. Now that these PRs have been merged to main, these changes no longer appear.